src/types/events/index.ts

```diff
@@ -1,5 +1,14 @@
 export * from './base-events';
-export { BotMessageEvent, GenericMessageEvent } from './message-events';
+export {
+  BotMessageEvent,
+  GenericMessageEvent,
+  MessageRepliedEvent,
+  MeMessageEvent,
+  MessageDeletedEvent,
+  ThreadBroadcastMessageEvent,
+  MessageChangedEvent,
+  EKMAccessDeniedMessageEvent,
```

Comment:
Is there any chance we can export all the message events? Why is the list limited to a subset of events?

Reply:
@jestrada Do you have anything missing in your mind?

Reply:
Hey @seratch, thanks for the reply! Ya, all of the Channel*MessageEvent types are missing which makes it impossible to refer to import them. Is it possible to export all message event types.

Reference: https://github.com/slackapi/bolt-js/blob/main/src/types/events/message-events.ts#L3

Reply:
Ah I got the point. This is an easy fix. Thanks for pointing this out.

Reply:
submitted a PR to fix this: https://github.com/slackapi/bolt-js/pull/1254

Reply:
Thanks @seratch!